### PR TITLE
Updates to Oceananigans 0.68

### PR DESCRIPTION
**Pull request by glwagner**
This PR updates `LESbrary.jl` to Oceananigans 0.68.

### CODE REVIEW

**Comment by glwagner** [comment]
I guess this supercedes #122

File: Project.toml [hunk 1]
```diff
@@ -25,7 +25,6 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Oceananigans = "^0.58"
```
Comment (navidcy) [inline Project.toml:28]: Oceananigans = "^0.68"

**Review by navidcy — APPROVED** [review]

File: Project.toml [hunk 2]
```diff
@@ -25,7 +25,6 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
```
Comment (navidcy) [inline Project.toml:27]:
```suggestion
[compat]
Oceananigans = "^0.68.2"
Oceanostics = "^0.6.3"
```

**Comment by navidcy** [comment]
Yeah!

**Pull request merged**